dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/BaseClasses.mo

```diff
@@ -248,7 +263,21 @@ partial model BaseGeneratorSynchronous "Synchronous machine - Base dynamic model
     Types.PerUnit cePu(start = Ce0Pu) "Electrical torque in p.u (base SNom/OmegaNom)";
     Types.PerUnit PePu(start = Ce0Pu*SystemBase.omega0Pu) "Electrical active power in p.u (base SNom)";
 
+
+    // Saturated mutual inductances and related variables
+    Types.PerUnit MdSatPPu(start = MdSat0PPu) "Direct axis saturated mutual inductance in p.u.";
+    Types.PerUnit MqSatPPu(start = MqSat0PPu) "Quadrature axis saturated mutual inductance in p.u.";
+    Types.PerUnit lambdaAirGapPu(start = LambdaAirGap0Pu) "Total air gap flux in p.u.";
+    Types.PerUnit lambdaADPu(start = LambdaAD0Pu) "Common flux of direct axis in p.u.";
+    Types.PerUnit lambdaAQPu(start = LambdaAQ0Pu) "Common flux of quadrature axis in p.u.";
+    Types.PerUnit mdsPu(start = Mds0Pu) "Direct axis saturated mutual inductance in the case when the total air gap flux is aligned on the direct axis in p.u.";
+    Types.PerUnit mqsPu(start = Mqs0Pu) "Direct axis saturated mutual inductance in the case when the total air gap flux is aligned on the quadrature axis in p.u.";
+    Types.PerUnit sin2Eta(start = Sin2Eta0) "Common flux of quadrature axis contribution to the total air gap flux in p.u.";
+    Types.PerUnit cos2Eta(start = Cos2Eta0) "Common flux of direct axis contribution to the total air gap flux in p.u.";
```

Comment:
This is a detail but all over the declarations and for init values, you first have direct axis values thus I would have had sin2Eta before cos2Eta